modules/test/playwright/tests/frontend-taglib-clay/main/alerts.spec.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../../fixtures/featureFlagsTest';
+import {clickAndExpectToBeHidden} from '../../../utils/clickAndExpectToBeHidden';
+import {claySamplePageTest} from './fixtures/claySamplePageTest';
+import {TabName} from './pages/ClaySamplePage';
+
+const test = mergeTests(
+	featureFlagsTest({
+		'LPS-178052': {enabled: true},
+	}),
+	claySamplePageTest
+);
+
+test.beforeEach('Select alerts tab', async ({claySamplePage}) => {
+	await claySamplePage.selectTab(TabName.ALERTS);
+});
+
+test('Verify alert contains all required attributes: icon, type text, and description.', async ({
+	claySamplePage,
+}) => {
+	await test.step('Check if the alert is present', async () => {
+		await expect(
+			claySamplePage.alertStatic('success').first()
+		).toBeAttached();
+	});
+
+	await test.step('Check if the alert displays status icon', async () => {
+		await expect(
+			claySamplePage
+				.alerStaticIcon('success', 'check-circle-full')
+				.first()
+		).toBeVisible();
+	});
+
+	await test.step('Check if the alert displays type text', async () => {
```

Comment:
This would be better described as, "Check if alert-success displays the correct lead text".